### PR TITLE
Feature questions (QuestionController)

### PR DESCRIPTION
**Pull request by aakash1692**

### CODE REVIEW

**Comment by vishivish18** [comment]
@aakashvermaee , First suggestion without even looking at the PR. Never raise PRs after such a long time with so many commits. Unless you are working on an entirely different feature. 😄

**Comment by aakash1692** [comment]
@vishivish18 😅 I'll keep that in mind

File: quiz-app-api/app/controllers/ResourceController.js [hunk 1]
```diff
@@ -1,71 +1,88 @@
...
+  update(req, res, next) {
+    console.log("this is the value of this: " + this)
+    console.log("update called");
+    res.send("update called from resource controller " + this)
+    next();
+  };
 
-    delete() {
+  delete() {
+    console.log("here")
```
Comment (vishivish18) [inline quiz-app-api/app/controllers/ResourceController.js]: @aakashvermaee But this does not have the Mongo `delete` function yet? Are you planning to add it?

Comment (aakash1692) [reply quiz-app-api/app/controllers/ResourceController.js]: Yes. I'll add it.

**Pull request merged**